cpp/log/cert.cc

```diff
@@ -233,8 +262,8 @@ string Cert::PrintTime(ASN1_TIME* when) {
 }
 
 
-Cert::Status Cert::IsIdenticalTo(const Cert& other) const {
-  return X509_cmp(x509_, other.x509_) == 0 ? TRUE : FALSE;
+StatusOr<bool> Cert::IsIdenticalTo(const Cert& other) const {
+  return StatusOr<bool>(X509_cmp(x509_, other.x509_) == 0 ? true : false);
```

Comment:
You're not safe from conversion between integer types, unfortunately. The following all compile, in a function with a `StatusOr<bool>`:

```C++
return true;
return 42;
return StatusOr<bool>(23);
return StatusOr<int>(77);
```

So there's no gain to be made?